extensions/wikidata/tests/src/org/openrefine/wikidata/qa/MockConstraintFetcher.java

```diff
@@ -211,10 +218,42 @@ public boolean hasDiffWithinRange(PropertyIdValue pid) {
         return true;
     }
 
-    public Map<PropertyIdValue, List<Value>> getParamConflictsWith(PropertyIdValue pid) {
-        Map<PropertyIdValue, List<Value>> propertyIdValueListMap = new HashMap<>();
-        List<Value> items = Arrays.asList(conflictingStatementValue, null);
-        propertyIdValueListMap.put(conflictingStatementPid, items);
-        return propertyIdValueListMap;
+    @Override
+    public Stream<Statement> getConstraintsByType(PropertyIdValue pid, String qid) {
+        EntityIdValue entityIdValue = Datamodel.makeWikidataItemIdValue("Q21502838");
+        PropertyIdValue propertyIdValue = Datamodel.makeWikidataPropertyIdValue("P2302");
+        Snak snak = Datamodel.makeValueSnak(propertyIdValue,entityIdValue);
+
+        PropertyIdValue property = Datamodel.makeWikidataPropertyIdValue("P2306");
+        Value propertyValue = Datamodel.makeWikidataPropertyIdValue("P31");
+        Snak snak1 = Datamodel.makeValueSnak(property,propertyValue);
+        List<Snak> group1 = new ArrayList<>();
+        group1.add(snak1);
```

Comment:
One way to make this slightly more concise is to use `Collections.singletonList` which creates a list with a single element.